### PR TITLE
fix(editorial): fix the automatic attribution role.

### PR DESCRIPTION
**Pull request by voidtek**
https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-41

### CODE REVIEW

File: profiles/common/modules/features/nexteuropa_editorial/src/Config.php [hunk 1]
```diff
@@ -41,6 +41,24 @@ public function isEditoralTeamMember($uid = 0) {
...
+   * @param int $nid
+   *    Node NID.
+   *
+   * @return bool
+   *    TRUE if the node is a team group, FALSE otherwise.
+   */
+  public function isEditoralTeamGroup($nid = 0) {
+    $is_team_member = FALSE;
+    $node = node_load($nid);
+    if (isset($node) && ($node->type == "editorial_team")) {
```
Comment (gillesdeudon) [inline profiles/common/modules/features/nexteuropa_editorial/src/Config.php]: @voidtek, that's why it is a suggestion only. ;-)

**Comment by gillesdeudon** [comment]
:+1

File: profiles/common/modules/features/nexteuropa_editorial/src/Config.php [hunk 2]
```diff
@@ -41,6 +41,24 @@ public function isEditoralTeamMember($uid = 0) {
...
+   *
+   * @param int $nid
+   *    Node NID.
+   *
+   * @return bool
+   *    TRUE if the node is a team group, FALSE otherwise.
+   */
+  public function isEditoralTeamGroup($nid = 0) {
+    $is_team_member = FALSE;
+    $node = node_load($nid);
```
Comment (imanoleguskiza) [inline profiles/common/modules/features/nexteuropa_editorial/src/Config.php]: I think this variable should be called 'is_editorial_team'

File: profiles/common/modules/features/nexteuropa_editorial/src/Config.php (hunk 1 above)
Comment (voidtek) [inline profiles/common/modules/features/nexteuropa_editorial/src/Config.php]: @gillesdeudon can you check that your request is done?

File: profiles/common/modules/features/nexteuropa_editorial/src/Config.php (hunk 2 above)
Comment (voidtek) [inline profiles/common/modules/features/nexteuropa_editorial/src/Config.php]: @imanoleguskiza Good point, I choose the name "is_team_group". Thanks.

**Comment by imanoleguskiza** [comment]
👍

**Comment by gillesdeudon** [comment]
:+1:

**Pull request merged**